src/doc.cpp

```diff
@@ -830,7 +830,8 @@ void Doc::CastOffDoc()
     Page *currentPage = new Page();
     CastOffPagesParams castOffPagesParams(contentPage, this, currentPage);
     CastOffRunningElements(&castOffPagesParams);
-    castOffPagesParams.m_pageHeight = this->m_drawingPageHeight - this->m_drawingPageMarginBot;
+    castOffPagesParams.m_pageHeight
+        = this->m_drawingPageHeight - this->m_drawingPageMarginTop - this->m_drawingPageMarginBot;
```

Comment:
Looks unrelated, doesn't it?

Reply:
The value passed is now `this->m_drawingPageContentHeight` and there is no need to deal with margins here anymore

Reply:
removed